src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl

```diff
@@ -51,7 +225,60 @@ function calc_hmax_latlon(elev, lon, lat, earth_radius)
     return hmax
 end
 
-function get_OGW_info(Y, orographic_info_rll)
+function compute_OGW_info(Y, elev_data, earth_radius)
+    # obtain lat, lon, elevation from the elev_data
+    nt = NCDataset(elev_data, "r") do ds
+        lon = ds["longitude"][:]
+        lat = ds["latitude"][:]
+        elev = ds["elevation"][:]
+        (; lon, lat, elev)
+    end
+    (; lon, lat, elev) = nt
+    FT = eltype(elev)
+
+    # compute hmax and hmin
+    hmax = calc_hmax_latlon(elev, lon, lat, earth_radius)
+
+    # compute χ
+    χ = calc_velocity_potential(elev, lon, lat, earth_radius)
+
+    # compute orographic tensor (t11, t21, t12, t22)
+    t11, t21, t12, t22 = calc_orographic_tensor(elev, χ, lon, lat, earth_radius)
+
+    topo_ll = (; hmax = hmax, hmin = FT(0.1) .* hmax, t11, t12, t21, t22)
```

Comment:
[Here](https://github.com/NOAA-GFDL/atmos_phys/blob/def83fee4a751e0d0762c1ae5159e13042129116/atmos_param/topo_drag/topo_drag.F90#L863) is the relationship between hmax and hpos (which we call hmax here) in the gfdl code. Would it be possible to add it here?

Reply:
I changed the function name to calc_hpoz_latlon and computed hmax from hpoz